services/bitbucket/bitbucket-pipelines.service.js

```diff
@@ -39,18 +46,7 @@ module.exports = class BitbucketPipelines extends BaseJsonService {
   }
 
   static render({ status }) {
-    const responses = {
-      SUCCESSFUL: { message: 'passing', color: 'brightgreen' },
-      FAILED: { message: 'failing', color: 'red' },
-      ERROR: { message: 'error', color: 'red' },
-      STOPPED: { message: 'stopped', color: 'yellow' },
-      EXPIRED: { message: 'expired', color: 'yellow' },
-      'never built': { message: 'never built', color: 'lightgrey' },
```

Comment:
at first I thought we missed the `never built` status in the status enumerations in `build-status.js`, but I think i'm wrong. This status will be handled exactly the same way since the `renderBuildStatusBadge` function will apply this label (since this doesn't exist in happy, nor orange, nor other lists) and the default `lightgrey` color right?

Reply:
Yes, exactly. 😃